static/org.openpsa.invoices/invoices.js

```diff
@@ -105,4 +105,10 @@ $(document).ready(function() {
             });
         });
     });
+
+    $('button.send_by_mail').on('click', function() {
+        var guid = $(this).attr('id').replace('invoice_', '');
+        var url = window.location.origin + '/invoices/invoice/action/send_by_mail/' + guid + '/';
```

Comment:
the `/invoices/` part in the URL is actually not part of the component, but comes from the DB structure, i.e. it can differ in different installations. We need to find some other way to build the URL here